.eslintrc.json

```
@@ -0,0 +1,22 @@
+{
+  "env": {
+    "es6": true,
```

```suggestion
    "es2021": true,
```
The parser couldn't handle some of the new(ish) JS syntax, but this seems to fix the majority of those errors.